libraries/Zigbee/examples/Zigbee_Color_Dimmer_Switch/Zigbee_Color_Dimmer_Switch.ino

```diff
@@ -145,6 +142,6 @@ void loop() {
   static uint32_t last_print = 0;
   if (millis() - last_print > 30000) {
     last_print = millis();
-    zbSwitch.printBoundDevices(Serial);
+    zbSwitch.printBoundDevices();
```

Comment:
Is this change correct? Shouldn't it have Serial as argument?

Reply:
Yes it should, without it will print to log_i instead. Fixed, idk how that was removed.